src/tests/end2end/BasicTests.cpp

```diff
@@ -35,8 +35,9 @@ TEST_P(BasicTests, BufferSetSubData) {
 }
 
 TEST_P(BasicTests, ReadPixelsTest) {
-    RGBA8 red(255, 0, 0, 255);
-    nxt::Buffer buffer = utils::CreateFrozenBufferFromData(device, &red, sizeof(red), nxt::BufferUsageBit::TransferSrc);
+    RGBA8 data[64];
+    data[0] = RGBA8(255, 0, 0, 255);
+    nxt::Buffer buffer = utils::CreateFrozenBufferFromData(device, data, sizeof(data), nxt::BufferUsageBit::TransferSrc);
```

Comment:
Nevermind. I realized if we do `((height - 1) * rowPitch + width) * depth` then this is fine